server/database/banano/known-accounts.json

```diff
@@ -938,6 +938,18 @@
         "owner": "ViteX",
         "type": "exchange"
     },
+    {
+        "address": "ban_3hfe17gwpi75ducquep38dcrnits9ffw36dhebuj4ej1adzzsxuq3b3mjcxz",
+        "alias": "ViteX Ban-001 Wallet",
+        "illicit": 0,
+        "owner": "ViNo",
+        "type": "Exchange"
```

Comment:
Thanks for the PR! Could you please update 'Exchange' to be the lowercase 'exchange'?